packages/api/portfolioDrafts/application/getApplicationStep.ts

```diff
@@ -51,3 +62,5 @@ export async function handler(event: APIGatewayProxyEvent): Promise<APIGatewayPr
     return DATABASE_ERROR;
   }
 }
+
+export const handler = middy(baseHandler).use(cors({ headers: "*", methods: "*" }));
```

Comment:
@juliusfitzhugh-ccpo and @zachclark-ccpo Can you coordinate on a place to put this CORS configuration? When the time comes to limit the headers/methods/origins, I'd like to be able to change it in one place rather than having to modify all our functions. So ideally, something like:

```ts
// TODO(AT-XXXX): Limit the origin(s) permitted by the CORS policy
export const CORS_CONFIGURATION: Type = {
  origin: "*",
  headers: "*",
  methods: "*",
}
```

Reply:
Adding on this branch. Coordinated with @zachclark-ccpo and we decided to add it to a new file `/api/utils/corsConfig.ts`.  Created an `Options` interface that the `cors` function expects.  I also added `AT-6712` for the TODO, is that the correct one or should it be added to a different ticket?

Reply:
That's great.  Looks like the right ticket.  I linked to the above in that ticket.